girder/api/rest.py

```diff
@@ -170,7 +170,8 @@ def retVal(user, token):
         return retVal(None, token)
     else:
         try:
-            ensureTokenScopes(token, TokenScope.USER_AUTH)
+            ensureTokenScopes(token, getattr(
+                cherrypy.request, 'requiredScopes', TokenScope.USER_AUTH))
```

Comment:
Do we need the fallback of `TokenScope.USER_AUTH` here? Every time `handleRoute` is called, `cherrypy.request.requiredScopes` should be getting set to something.